lib/tools/adb-commands.js

```diff
@@ -1953,6 +1953,16 @@ methods.takeScreenshot = async function takeScreenshot (displayId) {
   return stdout;
 };
 
+/**
+ * Return the result of 'adb shell dumpsys activity services <packagename>'
+ *
+ * @this {import('../adb.js').ADB}
+ * @returns {Promise<string>} the result of 'adb shell dumpsys activity services <packagename>'
+ */
+methods.getActivityService = async function getActivityService (pkgName) {
+  return await this.shell(['dumpsys', 'activity', 'services', pkgName]);
```

Comment:
should we somehow parse the output into a valid list? Otherwise this wrapper does not really do much

Reply:
Yea, initially i thought the command returned like list (part of output) below which was available via `dumpsys activity services` (no package name). 

```
Active foreground apps - user 0:
  #0: io.appium.settings
    mNumActive=1 mAppOnTop=false mShownWhileTop=true mShownWhileScreenOn=true
    mStartTime=-16m59s209ms mStartVisibleTime=-16m59s209ms
```

so something like `getActiveForegroundAppa`. (maybe we can create a new method for this later)

`dumpsys activity services io.appium.settings` had slightly better performance than `dumpsys activity services` (based on bash's `time`), so as the initial implementation, I made simple output with better performance way right now.

Reply:
I mean it makes sense to have the actual adb.shell call as a part of `isSettingsAppServiceRunningInForeground` method. We anyway don't use it anywhere else

Reply:
I see, makes sense